config.js

```diff
@@ -34,6 +34,14 @@ module.exports = {
   // See `bufferSize`.
   flushDelaySeconds: 30,
 
+  // If paths are present in this array, then these paths will be ignored before
+  // `samplingRate` based decisions are made. Paths must include a leading
+  // forward slash and be of the form:
```

Comment:
If you are not providing a regex, then your path must include a leading slash to satisfy the the condition on lib/tracing-policy.js#47. If you are using a regex there is no need to provide the leading slash. This is required so that if they specify an ignoreUrl of 'test/get' then we only want to ignore 'www.example.com/test/get' and not 'www.example.com/ns/test/get'. We could autofill the slash if not present but this would be ambiguous as it is not clear which of the urls the programmer intended. If they want both they should use a regex.